packages/atomic/src/components/atomic-search-box/atomic-search-box.tsx

```diff
@@ -430,6 +458,14 @@ export class AtomicSearchBox {
         e.preventDefault();
         this.focusPreviousValue();
         break;
+      case 'ArrowRight':
+        e.preventDefault();
+        this.focusPanel(this.listRefs[1]);
```

Comment:
I think we should only call this when `listRefs[1]` exists, like so:
```ts
this.listRefs[1] && this.focusPanel(this.listRefs[1])
```

Otherwise, I think we should change the type of `focusPanel` to something like:
```ts
private focusPanel(panel: HTMLElement | undefined)
```

Reply:
that is happening explicitly here
```
 if (this.panelInFocus === panel) {
      return;
    }
```
but weird that it doesn't complain that `listRefs[x]` could be `undefined`! I'll update